TodoApplication/Views/LoginView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  LoginView.swift
+//  TodoApplication
+//
+//  Created by Abhishek Kumar Sinha on 22/11/24.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+    @StateObject var  viewModel = LoginViewViewModel()
+   
```

Comment:
why state object ?

Reply:
# I used @StateObject to initialize and own the viewModel so it persists across view updates.